kodilla-spring/src/main/java/com/kodilla/spring/library/Library.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.spring.library;
+
+import org.springframework.stereotype.Service;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public final class Library {
+
+    private final List<String> books = new ArrayList<>();
+    private LibraryDbController libraryDbController;
+
+    public Library(final LibraryDbController libraryDbController) {
+        this.libraryDbController = libraryDbController;
+    }
+
+    public Library() {
+    }
```

Comment:
jesli uzyje tego kontruktora, to metody savaToDB i load.. beda rzucac null pointerem, bo libraryDbController bedzie zawsze nullem. Do wyrzucenia